packager/mpd/base/period.h

```diff
@@ -66,6 +66,12 @@ class Period {
     duration_seconds_ = duration_seconds;
   }
 
+  /// @return trickplay_cache.
+  const std::map<std::string, std::list<AdaptationSet*>> GetTrickPlayCache()
```

Comment:
You can name the function as |trickplay_cache| with the variable |trickplay_cache| renamed to |trickplay_cache_|.